languages/ruby/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,55 @@
+Ruby is a dynamic [object-oriented language]. Everything in Ruby is an [object][object].
+
+There are two ways to assign objects to names in Ruby - assigning variables or constants. Variables always start with lower-case letters and use [snake case][snake-case] for their formatting. A variable can be redefined to different objects over its lifetime. For example, `my_first_variable` can be defined and redefined many times using the `=` operator:
```

Comment:
```suggestion
There are a few ways to assign objects to names in Ruby - assigning  local variables, (we will refer to these generally as _variables_ or in the singular, _variable_) or constants. Variables always start with lower-case letters and use [snake case][snake-case] for their formatting. A variable can reference different objects over its lifetime. For example, `my_first_variable` can be defined and redefined many times using the `=` operator:
```

Reply:
@kotp I'm not sure I understand these changes. Help me understand pls :)
1) If we change "two" to "a few" we should provide more than two examples. I'm not sure what other ways are though?
2) Why the reference to `_local_` and the content in brackets? Why can't we just say "variables"?

Reply:
1. There are a few, we will show two.  (because the other ways don't really matter at this point)
2. Because there are instance variables, class variables, global variables.  The italics are for emphasis that we are talking about specific variables here.  They start with a lower case letter, others start with a sigil.  We don't have to mention it, but we can suggest that we are talking about a specific thing by emphasizing the term.

Reply:
Ok - I think this works:

```suggestion
There are two primary ways to assign objects to names in Ruby - using variables or constants. Variables are always written in [snake case][snake-case]. A variable can reference different objects over its lifetime. For example, `my_first_variable` can be defined and redefined many times using the `=` operator:
```

Reply:
Yes, I agree... keep the white lies to a minimum, but when necessary, keep them invisible as well.